common.go

```diff
@@ -1,4 +1,42 @@
 package sdk
 
+import (
+	"fmt"
+
+	"google.golang.org/grpc"
+)
+
 const NativeBin = "/opt/driver/bin/native"
 const NativeBinTest = "/opt/driver/src/build/native"
+
+const (
+	// MaxMsgSizeCLIName is name of the CLI flag to set max msg size.
+	MaxMsgSizeCLIName = "grpc-max-message-size"
+	// MaxMsgSizeCLIDesc is description for the CLI flag to set max msg size.
+	MaxMsgSizeCLIDesc = "max. message size to send/receive to/from clients (in MB)"
```

Comment:
Does this need to be exported?

Reply:
Oh, nevermind—I see this is for setting flag strings.

It is weird to put this into the SDK. This may be a bigger design question, so feel free to defer it, but if we are going to let libraries choose labels for command-line parameters, we shouldn't export the strings and force the caller to do the right thing—instead, we should give the library a function the caller can use to generate flags. 

But there is probably some established convention here, I'm still learning my way around.

Reply:
👍 makes sense, addressed by `MaxSendRecvMsgSizeMB(fs *flag.FlagSet)`